comparison/loadtest.go

```diff
@@ -232,21 +305,13 @@ func initLoadTest(t *terraform.Terraform, buildCfg BuildConfig, dumpFilename str
 		Value:   clearLicensesCmdValue,
 	}
 
-	resetBucketCmds := []localCmd{}
-	if s3BucketURI != "" && tfOutput.HasS3Bucket() {
-		deleteBucketCmd := localCmd{
-			msg:   "Emptying S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "rm", "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		prepopulateBucketCmd := localCmd{
-			msg:   "Pre-populating S3 bucket",
-			value: []string{"--profile", t.Config().AWSProfile, "s3", "cp", s3BucketURI, "s3://" + tfOutput.S3Bucket.Id, "--recursive"},
-		}
-
-		resetBucketCmds = []localCmd{deleteBucketCmd, prepopulateBucketCmd}
+	cfg, err := config.LoadDefaultConfig(context.Background(), config.WithSharedConfigProfile(t.Config().AWSProfile))
```

Comment:
@agarciamontoro in the future if we want to integrate with Github Actions this `profile` AWS access might not be eligible . We might need to perform a small refactor but it's simple

Reply:
Specifically we might need to use [config.LoadDefaultConfig](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials) to ensure stability across envs

Reply:
@toninis, do you mean using another profile? We are using `config.LoadDefaultConfig`, just using the profile configured by the user in the deployer.json file.